Parser.php

```diff
@@ -86,8 +88,14 @@ private function content2Data($content)
                 $data['playlist'][$mediaSequence]['isDiscontinuity'] = true;
             }
 
-            if (preg_match('/^#EXTINF:(.+),/', $line, $matches)) {
+            if (preg_match('/^#EXTINF:(\d+),(.+)/', $line, $matches)) {
                 $data['playlist'][$mediaSequence]['duration'] = +$matches[1];
+                $data['playlist'][$mediaSequence]['name'] = $matches[2];
+                continue;
+            }
+
+            if (preg_match('/^#EXTGRP:(.+)/', $line, $matches)) {
```

Comment:
could you please tell me what is EXTGRP tag about?

I can't find anything about it on the [HLS draft](https://tools.ietf.org/html/draft-pantos-http-live-streaming-20)